src/main/java/org/spongepowered/common/event/SpongeCommonEventFactory.java

```diff
@@ -676,16 +677,18 @@ public static NotifyNeighborBlockEvent callNotifyNeighborEvent(World world, Bloc
             return null;
         }
         try (CauseStackManager.StackFrame frame = Sponge.getCauseStackManager().pushCauseFrame()) {
-            final BlockState blockstate = (BlockState) ((net.minecraft.world.World) world).getBlockState(sourcePos);
+            final net.minecraft.world.chunk.Chunk chunk = ((WorldServer) world).getChunkFromBlockCoords(sourcePos);
```

Comment:
Why the difference here?

Reply:
I've just moved this from 
https://github.com/ImMorpheus/SpongeCommon/blob/115f88f4245b912e07b3040d0a6463c02858889c/src/main/java/org/spongepowered/common/event/SpongeCommonEventFactory.java#L699